internal/log/handlers/cli/cli.go

```diff
@@ -104,6 +104,9 @@ func logTable(w io.Writer, f log.Fields) error {
 // TypedLog is used for handling special "typed" logs to the CLI
 func (h *Handler) TypedLog(t string, e *log.Entry) error {
 	switch t {
+	case "engine":
+		fmt.Fprintf(h.Writer, "[engine] %s\n", e.Message)
+		return nil
```

Comment:
Yes this is good and how I was thinking it could be done.

The benefit is that we can also filter through these in the JSON based output for usage inside of the desktop app.

Reply:
Right, then I guess I can resolve this bit of the conversation.